src/Compressors/Compressor.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\DbDumper\Compressors;
+
+abstract class Compressor
```

Comment:
Let's simplify this. I think compressors only need to have one method `getCommand` which should return a string.

Having `level` and `extension` on a higher level is not needed. 

Also, because no implementation code will live here, change this from an abstract class to an interface.

Reply:
the ```getExtension``` method should go in the interface since it will need at Laravel-Backup